src/nataliia_chobu/Divide.java

```diff
@@ -0,0 +1,21 @@
+package nataliia_chobu;
+
+public class Divide {
+    public static void main(String[] args) {
+        System.out.println(divideWithoutDivisionOperator(12,3));
+    }
+
+    public static int divideWithoutDivisionOperator(int a, int b) {
+
+        int result = 0;
+
```

Comment:
I believe this method should be able work with all numbers. Try dividing -12 / 3 result will be 0 it is incorrect.